data-prepper-plugins/key-value-processor/README.md

```diff
@@ -78,6 +78,7 @@ When run, the processor will parse the message into the following output:
   * Default: `false`
   * Example: `remove_brackets` is `true`. `{"key1=(value1)"}` will parse into `{"key1": value1}`
   * Example: `remove_brackets` is `false`. `{"key1=(value1)"}` will parse into `{"key1": "(value1)"}`
+  * In the case of a key-value pair with a brackets and a split character, the behavior when `remove_brackets` is `true` will look like the following: `{key1=(value1&value2)}` will parse into `{"key1":"value1","value2)":null}`
```

Comment:
To rephrase it more accurately, the splitting will take priority over `remove_brackets=true`, e.g. `{key1=(value1&value2)}` will parse into `{"key1":"value1","value2)":null}` instead of `{"key1":"value1","value2":null}`